f.services.md

```diff
@@ -187,5 +187,11 @@ spec:
           access: 'true' # 'true' *needs* quotes in YAML, apparently
 ```
 
+```bash
+# Check the Network Policy is properly configured:
+kubectl run busybox --image=busybox --rm -it --restart=Never -- wget -O- http://nginx3:80                        # Ko
```

Comment:
Maybe `nginx3` should be `nginx`?
Also rephrase `Ko` to `This should not work`?

Reply:
Absolutely `nginx` and not `nginx3`, my bad because of a copy and paste from my bash, sorry for that :)
Also the rephrase is a better solution.

Reply:
Awesome, I’ll be more than happy to merge once you add these fixes! Thanks again for the contribution!